pocs/utils/rs232.py

```diff
@@ -12,37 +14,46 @@
 
 
 class SerialData(PanBase):
-    """
-    Main serial class
+    """SerialData wraps a PySerial instance for reading from and writing to a serial device.
+
+    Because POCS is intended to be very long running, and hardware may be turned off when unused
+    or to force a reset, this wrapper may or may not have an open connection to the underlying
+    serial device. Note that for most devices, is_connected will return true if the device is
+    turned off/unplugged after a connection is opened; the code will only discover there is a
+    problem when we attempt to interact with the device.
```

Comment:
If this is the case we should have a method on `pocs` or `observatory` that tests the connections and can be checked from the [ready](https://github.com/panoptes/POCS/blob/develop/pocs/state/states/default/ready.py) state. We can wait to do this when we write the methods that actually do the shutting down (probably in the [housekeeping](https://github.com/panoptes/POCS/blob/develop/pocs/state/states/default/housekeeping.py) state.)

Reply:
Submitted #185 to track this issue.